zerver/lib/bulk_create.py

```diff
@@ -116,6 +118,26 @@ def bulk_create_users(
 
     Subscription.objects.bulk_create(subscriptions_to_create)
 
+    full_members_system_group = UserGroup.objects.get(
+        name="@role:fullmembers", realm=realm, is_system_group=True
+    )
+    members_system_group = UserGroup.objects.get(
+        name="@role:members", realm=realm, is_system_group=True
+    )
+    group_memberships_to_create: List[UserGroupMembership] = []
+    for user_profile in profiles_to_create:
+        # All users are members since this function is only used to create bots
+        # and test and development environment users.
```

Comment:
Can we add an `assert` here about `user_profile.role` so that we don't get broken group memberships state if this function is ever called with this assumption broken?